src/AppInstallerCLICore/ExecutionReporter.cpp

```diff
@@ -100,6 +101,61 @@ namespace AppInstaller::CLI::Execution
         }
     }
 
+    bool Reporter::PromptForBoolResponse(Resource::LocString message, Level level)
+    {
+        const std::vector<PromptOption> options
+        {
+            PromptOption{ Resource::String::PromptOptionYes, true },
+            PromptOption{ Resource::String::PromptOptionNo, false },
+        };
+
+        auto out = GetOutputStream(level);
+        out << message << std::endl;
+
+        // Try prompting until we get a recognized option
+        for (;;)

```

Comment:
>for (;;) [](http://example.com/codeflow?start=8&length=8)

suggestion: maybe default to false if response is invalid? Infinite loop sort of inappropriate

Reply:
I can make it use a default if there is no response. But I don't know if we should also use it if the response is invalid in any other way. I tested a couple of other programs that do prompts like this (e.g. PS's `cp -Confirm`, `git add -p`), and they would loop if there was an invalid response.